src/go/rpk/pkg/cli/cmd/topic/consume.go

```diff
@@ -427,27 +427,30 @@ func unixMilli(nano int64) int64 { return nano / 1e6 }
 // The expressions below all end in (?::|$), meaning, a non capturing group for
 // the first time followed by a colon or the second time at the end.
 func parseConsumeTimestamp(
-	half string,
-) (length int, at time.Time, end bool, err error) {
+	half string, t0 ...time.Time,
```

Comment:
Can t0 be a required argument? And then below, where `len(t0) > 0`, we can use `!t0.IsZero()`. Variadic arguments aren't usually used for optional arguments.